exercises/poker/src/test/java/PokerTest.java

```diff
@@ -0,0 +1,181 @@
+import static org.junit.Assert.assertEquals;
+import java.util.Arrays;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class PokerTest
+{
+    @Test
+    public void OneHand()
+    {
+        final String hand = "4S 5S 7H 8D JC";
+        assertEquals(Arrays.asList(hand),Poker.bestHands(Arrays.asList(hand)));
+    }
+
+    @Test
+    @Ignore
+    public void NothingVsOnePair()
+    {
```

Comment:
To be consistent, maybe move this curly bracket to after the signature? So it becomes
`public void NothingVsOnePair() {` instead?